js/common.js

```diff
@@ -89,25 +89,6 @@ $(document).ready( function() {
         SetCookie("collapse_settings", t_cookie);
     });
 
-    $('input[type=text].autocomplete').autocomplete({
```

Comment:
Looks like there are fields on bug report / update pages that used the auto-complete functionality.  These are platform, OS, and OS_Build.  You can find them by searching for "autocomplete" in the code.  Were these working?  If so, then we should switch them to another approach.